### PR TITLE
Refactoring QuickOpenHTML, CSS, and JavaScript extensions

### PR DESCRIPTION
**Pull request by jacobsone**
Removing shared code from QuickOpenHTML, QuickOpenCSS, and QuickOpenJavaScript and moving code into a shared file.

### CODE REVIEW

File: src/search/QuickOpenHelper.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            return;
+        }
+        var fileLocation = selectedItem.fileLocation;
+
+        var from = {line: fileLocation.line, ch: fileLocation.chFrom};
+        var to = {line: fileLocation.line, ch: fileLocation.chTo};
+        EditorManager.getCurrentFullEditor().setSelection(from, to, true);
+    }
+
+    function itemSelect(selectedItem, query) {
```
Comment (petetnt) [inline src/search/QuickOpenHelper.js]: Needs JsDoc

File: src/brackets.js [hunk 2]
```diff
@@ -91,6 +91,7 @@ define(function (require, exports, module) {
     require("editor/CSSInlineEditor");
     require("project/WorkingSetSort");
     require("search/QuickOpen");
+	require("search/QuickOpenHelper");
```
Comment (petetnt) [inline src/brackets.js]: Nitpick: Align indent with others (I think this line has tabs instead of spaces)

File: src/search/QuickOpenHelper.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2012 Adobe Systems Incorporated. All rights reserved.
```
Comment (petetnt) [inline src/search/QuickOpenHelper.js]: Update copyright (Maybe 2012-present like in `License.md`)

**Comment by petetnt** [comment]
Made few comments otherwise LGTM :+1:

File: src/search/QuickOpenHelper.js (hunk 3 above)
Comment (redmunds) [inline src/search/QuickOpenHelper.js]: The copyright date for new files should be the year that they were created, so 2016 in this case.

Comment (petetnt) [inline src/search/QuickOpenHelper.js]: @redmunds whoops.

Comment (abose) [inline src/search/QuickOpenHelper.js]: The new copyright notices has been changed to the form `2016 - present`. Please update the date in the format.

**Comment by abose** [comment]
@jacobsone  Thank you for contributing to Brackets. You will need to Accept the [Brackets CLA](http://dev.brackets.io/brackets-contributor-license-agreement.html) before we can merge this pull request.
For discussions with the brackets community, please join the [brackets slack channel](https://brackets.slack.com/) by sending a mail to admin@brackets.io with the subject line `slack registration request` specifying the email addresses you would like to register.
[brackets dev google groups link here](https://groups.google.com/forum/#!forum/brackets-dev) .

**Comment by MarcelGerber** [comment]
LGTM too. Thank you!
Merging.

**Pull request merged**